styles/styles.css

```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
   font-size: 16px;
 }
 
+/* REDESIGN Buttons */
+.redesign-v2 a.button.primary,
+.redesign-v2 button.primary,
+.redesign-v2 a.button.secondary,
+.redesign-v2 button.secondary,
+.redesign-v2 a.button.tertiary,
+.redesign-v2 button.tertiary {
+  align-items: center;
```

Comment:
Can’t we add a button class to a button? I think these selectors can be way cleaner